src/Pos/Types/Arbitrary.hs

```diff
@@ -42,14 +42,8 @@ import           Pos.Constants              (epochSlots, sharedSeedLength)
 import           Pos.Core.Address           (makePubKeyAddress, makeRedeemAddress,
                                              makeScriptAddress)
 import           Pos.Core.Coin              (coinToInteger, divCoin, unsafeSubCoin)
-import           Pos.Core.Types             (Address (..), ChainDifficulty (..), Coin,
-                                             CoinPortion, EpochIndex (..),
-                                             EpochOrSlot (..), LocalSlotIndex (..),
-                                             SharedSeed (..), SlotId (..), Timestamp (..),
-                                             getCoinPortion, mkCoin,
-                                             unsafeCoinPortionFromDouble, unsafeGetCoin)
-import           Pos.Core.Types             (ApplicationName (..), BlockVersion (..),
-                                             SoftwareVersion (..))
+import qualified Pos.Core.Types             as Types
```

Comment:
1. It's ok to change some imports to qualified if they are used one or two times in this file. It's not the case for `Coin`.
2. No, there will be no warning about double import. Add it to
> `import           Pos.Core.Types             (BlockVersion (..), SoftwareVersion (..))`

(which is one line below)